cmake/mod_bundled_packages_json.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+#
+# This script removes the given package entries in the bundled_packages JSON files. This is used by the PluginInstall
+# CMake module.
+#
+
+import argparse
+import collections
+import json
+import os
+import sys
+
+
+def find_json_files(work_dir: str) -> list:
+    """
+    Finds all JSON files in the given directory recursively and returns a list of those files in absolute paths.
+    :param work_dir: The directory to look for JSON files recursively.
+    :return: A list of JSON files in absolute paths that are found in the given directory.
+    """
+    json_file_list = []
+    for root, dir_names, file_names in os.walk(work_dir):
+        for file_name in file_names:
+            abs_path = os.path.abspath(os.path.join(root, file_name))
+            json_file_list.append(abs_path)
+    return json_file_list
+
+
+def remove_entries_from_json_file(file_path: str, entries: list) -> None:
```

Comment:
```suggestion
def remove_entries_from_json_file(file_path: str, entries: List[str]) -> None:
```
The documentation states that it's a list of strings, so might as well make the typing agree to that, right?

Reply:
This is done on purpose. `List` requires the `typing` package. Because this is kinda a standalone script, I made it this way so it doesn't have extra dependencies.

Reply:
But the `typing` package is required to be built-in in Python since 3.5, so the same could be said for the `os` package or the `argparse` package.